### DISCUSSION TITLE
About notifications of releases of freac

### DISCUSSION THREAD

**Discussion by RafaelLinux** [enzo1982/freac#491] [General Discussion]
I'm subscribed to freac releases notifications and got "Continuous build" alerts. However, I tried to find what's new on those releases but didn't find a file where users can know what changes or new features and didn't find it. There is such file?

Thank you

**Comment by enzo1982** [comment]
The [Readme file](https://github.com/enzo1982/freac/blob/master/Readme#L164) lists the important user-visible changes since the last official release.

If you want to see the changes from one continuous build to another, you can check the [history of the Readme file](https://github.com/enzo1982/freac/commits/master/Readme).

You can also check the [history of the repository](https://github.com/enzo1982/freac/commits/master) to see all changes, including those that are not listed in hte Readme file.

**Reply by RafaelLinux** [reply]
Perfect!! I was unaware of that possibility. I will keep an eye on it from now on.
Thank you
